src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: '0',
+    };
+  }
+
+  render() {
+    const { result } = this.state;
+    return (
+      <div>
+        <Display result={result} />
+        <ButtonPanel />
+      </div>
```

Comment:
- [ ]  Also, please wrap the children components in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))`